src/app/examples/button-example/button-example.component.html

```diff
@@ -1,8 +1,13 @@
-<button kirby-button>Kirby Button</button>
+<kirby-card [themeColor]="themeColor">
```

Comment:
Reverse the order of the examples:
1. Regular buttons
2. Buttons with attention level inside cards
Also add h2 headings:
"Standalone button"
"Button with attention level"